Examples/Liquidate/DummyERC20A.sol

```diff
@@ -0,0 +1,55 @@
+// Represents a symbolic/dummy ERC20 token
+
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+contract DummyERC20A {
+    uint256 t;
```

Comment:
We should use full names. `t` is `totalSupply`, `a` is `allowances`, `b` is `balances` etc.

Reply:
next time.... and in the template folder

Reply:
is it hard to change?